src/main/java/io/github/ryotan/code/rs/inject/CodeParamConverterProvider.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code.rs.inject;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.ParamConverterProvider;
+import javax.ws.rs.ext.Provider;
+
+import io.github.ryotan.code.util.CodeEnumReflectionUtil;
+
+@Provider
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Type Javadoc comment is missing an null tag.](https://www.codacy.com/app/ryotan/code-enum-support/file/2976069082/issues/source?bid=3375378&fileBranchId=3375797#l19)